### PR TITLE
changed LibreELEC URLs in (mainly) descriptions to https://libreelec.tv

### PR DESCRIPTION
**Pull request by jo3rg**

### CODE REVIEW

File: config/noobs/partition_setup.sh [hunk 1]
```diff
@@ -29,7 +29,7 @@ md5sumCheck() {
       echo "# LibreELEC failed md5 check - Installation will quit #"
       echo "#                                                     #"
       echo "#    Your original download was probably corrupt.     #"
-      echo "#  Please visit www.libreelec.tv and get another copy #"
+      echo "#  	Please visit libreelec.tv and get another copy 	  #"
```
Comment (InuSasha) [inline config/noobs/partition_setup.sh]: Whitespaces mistake (tab?)

Comment (jo3rg) [inline config/noobs/partition_setup.sh]: @InuSasha -> no, this seems to be a github diff visualisation issue as you can see in the full file: https://github.com/jo3rg/LibreELEC.tv/blob/changed_urls/config/noobs/partition_setup.sh#L32

Comment (InuSasha) [inline config/noobs/partition_setup.sh]: no there is tab, exact in front of the P.
test with vi on my machine

Comment (CvH) [inline config/noobs/partition_setup.sh]: very hard to spot

**Comment by chewitt** [comment]
No objections on the content but please squash changes to a single commit with a brief commit message. Thanks.

**Comment by MindTooth** [comment]
Thank you :-)

**Comment by jo3rg** [comment]
@chewitt & @MindTooth done

**Pull request merged**